paddle/fluid/operators/optimizers/dpsgd_op.h

```diff
@@ -79,7 +79,10 @@ class DpsgdOpKernel : public framework::OpKernel<T> {
     float X;
     float mu = 0.0;
     float U1, U2;
-    unsigned seed = (unsigned int)(time(NULL));
+    unsigned seed = ctx.Attr<int>("seed");
```

Comment:
Not important, you don't have to change if CI waiting time is long:

1. "unsigned" is same to "unsigned int" in c & c++, you can write (unsigned) at line 84

2. Remove useless code comment between line 90 - 94?

Reply:
done